lifecycle/lifecycle/auth/authorize.py

```diff
@@ -207,60 +207,60 @@ def list_permitted_families(
             return all_families
 
         if permission.fatman_family is not None:
-            family_names.add(permission.fatman.name)
+            family_names.add(permission.fatman_name)
 
     return [name_to_family[name] for name in sorted(family_names)]
 
 
 @db_access
 def grant_permission(
     auth_subject: models.AuthSubject,
-    fatman_name: Optional[str],
-    fatman_version: Optional[str],
+    job_name: Optional[str],
+    job_version: Optional[str],
     scope: str,
 ):
     """
-    Grant permission to access the resource (fatman, fatman family) within requested scope
-    :param fatman_version: Exact fatman version or an alias ("latest" or wildcard)
+    Grant permission to access the resource (job, job family) within requested scope
+    :param job_version: Exact job version or an alias ("latest" or wildcard)
     """
     subject_filter = Q(auth_subject=auth_subject)
     scope_filter = Q(scope=scope)
-    if fatman_name and fatman_version:
-        resource_filter = Q(fatman__name=fatman_name, fatman__version=fatman_version)
-    elif fatman_name:
-        resource_filter = Q(fatman_family__name=fatman_name, fatman__isnull=True)
+    if job_name and job_version:
+        resource_filter = Q(fatman__name=job_name, fatman__version=job_version)
+    elif job_name:
+        resource_filter = Q(fatman_family__name=job_name, fatman__isnull=True)
     else:
         resource_filter = Q(fatman_family__isnull=True, fatman__isnull=True)
 
     queryset = models.AuthResourcePermission.objects.filter(
         subject_filter & resource_filter & scope_filter
     )
     if queryset.exists():
-        logger.warning(f'Permission for {auth_subject} to {fatman_name} {fatman_version} (scope {scope}) is already granted')
+        logger.warning(f'Permission for {auth_subject} to {job_name} {job_version} (scope {scope}) is already granted')
         return
 
-    if fatman_name and fatman_version:
-        fatman_model = models.Fatman.objects.get(name=fatman_name, version=fatman_version)
+    if job_name and job_version:
+        job_model = models.fatman.objects.get(name=job_name, version=job_version)
         permission = models.AuthResourcePermission(
             auth_subject=auth_subject,
-            fatman=fatman_model,
+            job=job_model,
```

Comment:
This is the name of the database column, we'll take care of it later on
```suggestion
            fatman=job_model,
```